apps/io-services-cms-webapp/src/utils/converters/__tests__/service-lifecycle-converters.test.ts

```diff
@@ -1,12 +1,46 @@
-import { test, expect } from "vitest";
+import { test, expect, describe } from "vitest";
 import { ServiceLifecycle } from "@io-services-cms/models";
-import { toServiceStatus } from "../service-lifecycle-converters";
+import {
+  itemToResponse,
+  payloadToItem,
+  toServiceStatus,
+} from "../service-lifecycle-converters";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { ServicePayload } from "../../../generated/api/ServicePayload";
+import { FiscalCode } from "../../../generated/api/FiscalCode";
 
 const fsm: ServiceLifecycle.ItemType["fsm"] = {
   state: "rejected",
 };
 
-test("test service-lifecycle-converters", () => {
-  const result = toServiceStatus(fsm);
-  expect(result.reason).not.toBeDefined();
+describe("test service-lifecycle-converters", () => {
+  test("test service-lifecycle-converters", () => {
+    const result = toServiceStatus(fsm);
+    expect(result.reason).not.toBeDefined();
+  });
+
+  test("Authorized User should contains the default fiscal code and the one on request", () => {
+    const aNewService = {
+      name: "a service",
+      description: "a description",
+      organization: {
+        name: "org",
+        fiscal_code: "00000000000",
+      },
+      metadata: {
+        scope: "LOCAL",
+      },
+      authorized_recipients: ["BBBBBB99C88D555I"],
+    } as unknown as ServicePayload;
+
+    const result = payloadToItem(
+      "ffefefe" as NonEmptyString,
+      aNewService,
+      "AAAAAA00A00A000A" as FiscalCode
```

Comment:
Define a const (i.e. aSandboxFiscalCode) and use it instead of repeat string value two times

Reply:
[4aa94c7](https://github.com/pagopa/io-services-cms/pull/115/commits/4aa94c77cc80936cffcf93745d06bba979cb9d27)